ci-operator/config/cspi-qe/oadp-qe-automation/cspi-qe-oadp-qe-automation-main.yaml

```diff
@@ -0,0 +1,82 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  project_image:
+    dockerfile_literal: |
```

Comment:
Why are we using dockerfile_literal here instead of creating a custom image from a dockerfile.

When we use dockerfile_literal in a config file we are essentially saying that this particular dockerfile is only for this one config file. It will not be able to be used by anything else in OCP CI without directly duplicating it into another config file.